### PR TITLE
Rename Democratic Republic of the Congo

### PR DESCRIPTION
**Pull request by mgrassotti**
Trello: https://trello.com/c/HOnUP71d/493-democratic-republic-of-the-congo-fixes-medium

Rename the country slug

This is related to https://github.com/alphagov/whitehall/blob/19cd7d72de32454d532c195f35b027fa1b3ba6ac/db/data_migration/20160505110730_redirect_worldwide_priorities.rb

### CODE REVIEW

File: db/data_migration/20161007111802_setup_redirects_for_drc.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+new_path = "/government/world/democratic-republic-of-the-congo"
+old_paths = [ "/government/world/democratic-republic-of-congo",
+              "/government/priority/increasing-business-with-the-democratic-republic-of-congo",
+              "/government/priority/supporting-british-nationals-in-the-drc",
+              "/government/priority/supporting-conflict-resolution-and-civilian-protection-in-the-democratic-republic-of-congo" ]
+
+old_paths.each do |old_path|
+  redirects = [{ path: old_path, destination: new_path, type: "exact" }]
+  PublishingApiRedirectWorker.perform_async(old_path, redirects, I18n.default_locale.to_s)
```
Comment (boffbowsh) [inline db/data_migration/20161007111802_setup_redirects_for_drc.rb]: This takes a `content_id` not a path as its first argument

Comment (mgrassotti) [inline db/data_migration/20161007111802_setup_redirects_for_drc.rb]: 👍 I didn't notice this was updated

Comment (mgrassotti) [inline db/data_migration/20161007111802_setup_redirects_for_drc.rb]: I updated the code coherently

**Comment by mgrassotti** [comment]
👍 I will remove the redirects from here and create a PR on router-data

**Comment by mgrassotti** [comment]
https://github.gds/gds/router-data/pull/574

**Pull request merged**